tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+cookiecutter==0.9.0
+coverage==3.7.1
+ddt==0.8.1
+diff-cover==0.7.2
+Django==1.4.16
+django_nose==1.2
+fs==0.5.0
+lazy==1.2
+mock==1.0.1
+nose==1.3.4
+nose-parameterized==0.3.5
+pep8==1.5.7
+pylint==0.28
+pypng==0.0.17
+rednose==0.4.1
+requests==2.4.3
+selenium==2.44.0
+simplejson==3.6.5
+unicodecsv==0.9.4
+webob==1.4
```

Comment:
@Kelketek You're right, it's good to test with the most recent versions often. I believe the Travis CI integration Braden is working on will be great to catch problems with never versions. If you look at the list of frozen dependencies, you'll see that more than half (`coverage`, `ddt`, ...) are test-only dependencies that will never get installed to production systems. Figuring out why your tests won't run only to realize it's because a backwards-incompatible `rednose` package was released is a waste of time.